.github/workflows/release.yml

```
@@ -9,26 +9,26 @@ jobs:
     outputs:
       config_package: ${{ steps.config_package.outputs.configPackage }}
     steps:
-      - name: Ensure that required repository variable has been created for the Package
-        id: config_package
+      - id: config_package
+        name: Ensure that required repository variable has been created for the Package
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:32: Double quote to prevent globbing and word splitting

(shellcheck)